data/persons.txt

```diff
@@ -1850,3 +1849,174 @@ ship "Modified Ladybug"
 	explode "huge explosion" 10
 	"final explode" "final explosion large"
 	description `This is an old war-torn Hai Ladybug with numerous modifications done on the hull. Apart from the additional turret mounts, the main adjustments on the vessel are three huge engine cells - two in the back replacing the old engine pods, and one in the front replacing the gun mounts. These push the Modified Ladybug to speeds only an insane captain would enjoy.`
+
+
+person "Patrol Team"
+	government "Author"
+	frequency 300
+	personality
+		mining harvests plunders
+	phrase
```

Comment:
I think we should have a limited amount of quotes per individual person. This is a `pers` representing the Reviewer team as a whole. We can do some trickery with phrases to make sure each individual's quotes are equal weight when being displayed, but I still feel we should keep quotes from specific people at a certain amount, the more we add the less of a chance each individual quote has to show, especially considering `persons` are rare to begin with. I feel like maybe around 4-8 per person or something like that would be fine, but I am open to discussion on that.

Also keep in mind, since this is a `pers` for the Reviewer team in general and not one individual, that means either future interested Reviewers or those who become Reviewers in the future may want their own quotes, hence why I think a lower amount per person is fair and workable.